### ISSUE TITLE
A/D Converter for speaker in Nitrokey

### ISSUE THREAD

**Issue by nitrosimon** [Nitrokey/nitrokey-documentation#325]
Explicitly mention all Nitrophones contain an A/D converter which prevents the speakers being used as a microphone

**Comment by daringer** [comment]
yup, faq entry in nitropads (mention which models exactly do have one)
